src/ColorContextMenu-Core.package/RadialMenuItemMorph.class/instance/update..st

```diff
@@ -0,0 +1,4 @@
+as yet unclassified
+update: aSymbol
+
+	aSymbol = #color ifTrue: [self updateLabelColor]
```

Comment:
In most cases, it is an anti-pattern to use observer on yourself. Can't you simply override the `#color:` setter in this case? Also affects the changes in `#privateInteractionState:` and `#initialize`.

Reply:
done